util/docker_cmd.sh

```diff
@@ -37,13 +38,21 @@ fi
 # IF we are using docker on non Linux and docker-machine isn't working print an error
 # ELSE set usb_args
 if [ ! "$(uname)" = "Linux" ] && [ "$RUNTIME" = "docker" ] && ! docker-machine active >/dev/null 2>&1; then
-    errcho "Error: target requires docker-machine to work on your platform"
-    errcho "See http://gw.tnode.com/docker/docker-machine-with-usb-support-on-windows-macos"
-    exit 3
+	errcho "Error: target requires docker-machine to work on your platform"
+	errcho "See http://gw.tnode.com/docker/docker-machine-with-usb-support-on-windows-macos"
+	exit 3
 else
-    usb_args="--privileged -v /dev:/dev"
+	usb_args="--privileged -v /dev:/dev"
+fi
+
+qmk_firmware_dir=$(pwd -W 2>/dev/null) || qmk_firmware_dir=$PWD  # Use Windows path if on Windows
+qmk_userspace_dir=""
+userspace_docker_args=""
+
+if [ -n "$(qmk userspace-path)" ]; then
```

Comment:
Previously it had been possible to use the docker build commands without having the QMK CLI installed on the host.  Should we add some way to support such configuration even in the presence of userspace (e.g., by picking up `$QMK_USERSPACE` from the environment before trying to discover it through `qmk`)?

Reply:
Done.